src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -128,14 +136,22 @@ cd ${BUILD_HOME}
 git clone --single-branch --depth=1 -b ${JAVACPP_BRANCH_TAG} ${JAVACPP_BRANCH}
 cd javacpp-presets
 
-${MAVEN_PATH} clean install --projects .,tritonserver
-${MAVEN_PATH} clean install -f platform --projects ../tritonserver/platform -Djavacpp.platform=linux-x86_64
+# Remove developer_tools/server related build
+if [ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -eq 1 ]; then
+    rm -r tritonserver/src/gen
+    rm tritonserver/src/main/java/org/bytedeco/tritonserver/presets/tritondevelopertoolsserver.java
+fi
+
+mvn clean install --projects .,tritonserver
+mvn clean install -f platform --projects ../tritonserver/platform -Djavacpp.platform=linux-x86_64
 
 # Copy over the jar to a specific location
 mkdir -p ${JAR_INSTALL_PATH}
 cp ${BUILD_HOME}/javacpp-presets/tritonserver/platform/target/tritonserver-platform-*shaded.jar ${JAR_INSTALL_PATH}/tritonserver-java-bindings.jar
 
-rm -r ${BUILD_HOME}/javacpp-presets/
-rm -r /root/.m2/repository
+if [ ${KEEP_BUILD_DEPENDENCIES} -eq 1 ]; then
```

Comment:
This looks like the reverse. Shouldn't this be if it's not equal to 1, then delete?

Reply:
We have been pretty bad at following the guidelines for [bash scripting](https://stackoverflow.com/questions/2933843/why-0-is-true-but-false-is-1-in-the-shell). I would advocate for `True == 0` instead of `True == 1`

Reply:
Great find! Let's start a conversation offline about this. This is news to me. I don't want to block this PR, but I think we'd want to decide this on a team level and have a PR to standardize all of our tests. Otherwise, it would be inconsistent and could lead to confusion/problems.

Reply:
Since this change has been propagated to the javacpp side, can we do this in a separate PR?

Reply:
^ @dyastremsky

Reply:
Okay, but please let's do it ASAP. We do not want this getting forgotten and us being inconsistent.